test/run_tests.sh

```diff
@@ -248,7 +287,7 @@ function run_xla_op_tests3 {
   run_test "$CDIR/spmd/test_xla_auto_sharding.py"
   run_test "$CDIR/spmd/test_spmd_parameter_wrapping.py"
   run_test "$CDIR/spmd/test_mp_input_sharding.py"
-  run_test "$CDIR/spmd/test_spmd_lowering_context.py"
+  run_save_tensor_hlo "$CDIR/spmd/test_spmd_lowering_context.py"
```

Comment:
Can we add this test in `test/tpu/run_tests.sh` too? Without TPU execution, I'm not sure we're exercising any SPMD feature end-to-end.

Reply:
I created a new commit to handle it, since we require the dumped IR graphs there.